spec/system/admin/invoice_print_spec.rb

```diff
@@ -548,8 +548,9 @@
           expect(page).to have_content "($500,150.00 / kg)" # unit price
           expect(page).to have_content "3 1g $416.79 $1,250.37 20.0% $1,500.45"
           # Enterprise fee
-          expect(page).to have_content "#{enterprise_fee.name} fee by $104.35 15.0% $120.00"
-          expect(page).to have_content "coordinator #{user1.enterprises.first.name}"
+          expect(page).to have_content(
+            "#{enterprise_fee.name} fee by coordinator $104.35 15.0% $120.00 #{distributor.name}"
```

Comment:
This matches what's in the locale file... https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/en.yml#L3289

Reply:
Hmm, I wonder why the spec passed before, that looks different.